portfolio/src/main/java/com/google/sps/servlets/UserSubmittedCommentsServlet.java

```diff
@@ -32,40 +32,59 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/map")
-public class MapServlet extends HttpServlet {
+@WebServlet("/submitted-comments")
```

Comment:
I tried very hard to accept this name - "submitted-comments". But every time I read it, it confuses me. Because it's not a comment. It's a location user submitted on map.

Reply:
+1

Reply:
Edited to 'submitted-locations"